### ISSUE TITLE
`crc start` should wait until `OpenShift status` in `crc status` is not `stopped`

### ISSUE THREAD

**Issue by savitaashture** [code-ready/crc#394]
1. Installed CRC successfully
2. oc login
```
oc login -u kubeadmin -p 9gZtH-EWZLT-mZNFP-bkKwm https://api.crc.testing:6443
Error from server (InternalError): Internal error occurred: unexpected response: 503
```

/cc @anjannath

**Comment by anjannath** [comment]
@savitaashture I am not able to reproduce this locally, can you try to reproduce this on your side with logging enabled, and paste the output, that might have more info.
```
oc login -u kubeadmin -p 9gZtH-EWZLT-mZNFP-bkKwm https://api.crc.testing:6443 -v=8
```

**Comment by praveenkumar** [comment]
This means that the auth operator is not working as expected so might be wait for some time and then try again.

**Comment by savitaashture** [comment]
@anjannath currently i deleted and recreated so not facing that issue.
But if at all i  issue reproduced again i will execute the command which you mentioned and paste the output.

@praveenkumar i waited for 10-15 min it dint work

**Comment by karstengresch** [comment]
Had the same problem and found out that OpenShift wasn't ready yet (startup takes around 6 minutes, even on my [superfast machine](https://www.msi.com/Laptop/GT75-Titan-8SX)).

Since then, I check with 

```crc status```

and never got this problem again...

**Comment by cfergeau** [comment]
I've changed this bug title. It's indeed unexpected that `crc start` says it's waiting for the cluster to be ready, but `crc status` says the opposite after `crc start` ended. `crc start` should use the same condition as `crc status` to decide the cluster is up and running. The relevant function call in `machine.go` is `operatorsRunning, err := oc.GetClusterOperatorStatus(ocConfig)`

**Comment by cfergeau** [comment]
See also https://github.com/code-ready/crc/issues/237

**Comment by cfergeau** [comment]
See https://github.com/code-ready/crc/pull/242 also

**Comment by gbraad** [comment]
I'd rather leave this issue closed as this is a dup of #237
